src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestStageTest.java

```diff
@@ -225,6 +231,8 @@ public void testRefreshSnowflakeMetadataRemote() throws Exception {
         "foo/streaming_ingest/", metadataWithAge.fileTransferMetadata.getStageInfo().getLocation());
     Assert.assertEquals(
         "placeholder", metadataWithAge.fileTransferMetadata.getPresignedUrlFileName());
+
+    Assert.assertEquals(123, parameterProvider.getMaxChunkSizeInBytes());
```

Comment:
https://github.com/snowflakedb/snowflake-ingest-java/pull/118#discussion_r791276629 @sfc-gh-tzhang here's the test